content/en/docs/how-tos/add-jobs-to-testgrid.md

```diff
@@ -0,0 +1,43 @@
+---
+title: "Testgrid"
+date: 2020-10-28T11:14:39-04:00
+draft: false
+---
+
+We talk about two distinct components in this doc - testgrids and release-informing/blocking/broken jobs. 
+All the release-informing/blocking/broken jobs jobs end up in the testgrid but we can also have jobs which are not release-informing/blocking/broken jobs ending up in the testgrid if they are added to the `/release/core-services/testgrid-config-generator/_allow-list.yaml`.
+
+## What are testgrids?
+Testgrids are used to measure the health of the application by displaying the results of jobs which serve as an early warning system giving us an indication of the health of the system as a whole and what features might have been affected by a release.
+
+## What is a release-informing/blocking/broken job?
+* `Blocking jobs` are those that signal widespread failure of the platform. These are traditionally the core end-to-end test runs on our major platforms and upgrades from previous versions. 
```

Comment:
```suggestion
* Blocking jobs are those that signal widespread failure of the platform. These are traditionally the core end-to-end test runs on our major platforms and upgrades from previous versions. 
```

Reply:
These are not code concepts and should not be in `code blocks`

Reply:
These jobs also do not signal anything widespread - an individual end-to-end test that has a very targeted focus could fail and indicate nothing widespread. These are jobs that expose our conformance criteria.